app/gui/src/project-view/components/widgets/FileBrowserWidget.vue

```diff
@@ -212,6 +213,26 @@ Promise.all([currentUser.promise.value, currentOrganization.promise.value]).then
         </div>
       </TransitionGroup>
     </div>
+    <div v-if="writeMode" class="fileNameBar">
+      <input
+        v-model="fileName"
+        class="fileNameInput"
+        @pointerdown.stop
+        @click.stop
+        @contextmenu.stop
+        @keydown.backspace.stop
+        @keydown.delete.stop
+        @keydown.arrow-left.stop
+        @keydown.arrow-right.stop
+        @keydown.enter="acceptCurrentFile() === false || $event.stopPropagation()"
```

Comment:
Should this be conditional? If the filename box is focused, I don't think I would expect Enter to be handled by the parent handler if the filename is invalid (empty).